### ISSUE TITLE
Feature request: See through mode

### ISSUE THREAD

**Issue by StrikeNP** [googlevr/cardboard#23]
I'm currently starting a project where I'm building a sort of AR/VR viewer where the primary display is the camera output. While I'm unfamiliar with the Cardboard development environment, I can see that see-through mode was an (experimental) feature of the previous iteration of Cardboard VR SDK. I'd find it useful if that feature was brought into the new api, however in the meantime I'll be looking for alternative workarounds.

**Comment by jballoffet** [comment]
Thank you for requesting this. See-through was a GVR feature which only worked on Lenovo Mirage Solo, not Cardboard. In order to be able to generate a 3D background you will need a device with two cameras separated from each other by a known distance. With a single camera, like on phones, you can generate a flat see-through effect by rendering its input to a 3D plane in the scene.

**Comment by StrikeNP** [comment]
My apologies, I did not realize it was a GVR feature. I understand 2 cameras is needed for 3D, but for my use case, a 2D surface is sufficient. 
I'm currently attempting to teach myself OpenGL and Android NDK basics to give me a better foundation for implementing stuff myself. Even if this feature request doesn't go through, I'd love to hear any resource recommendations people may have for building a foundation to work with the Cardboard SDK under. The new cardboard site is a bit... bare bones.

**Comment by sakdavong** [comment]
That's not true at all... See through was working with other phone (I did it with another Tango Phone for example) and it is implemented by many application.
You won't do it OK, but don't tell it's no feasible... Many applications are doing it (for example, drone FPS view when showing you the real environment with normal cardboard and phones apps.... Freeflight6 from Parrot is one of them).
